etcdctlv3/command/put_command.go

```diff
@@ -31,9 +31,31 @@ var (
 // NewPutCommand returns the cobra command for "put".
 func NewPutCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "put",
+		Use:   "put [options] <key> <value>",
 		Short: "Put puts the given key into the store.",
-		Run:   putCommandFunc,
+		Long: `
+When <value> has first character of '-', it is interpreted as a flag.
+Insert '--' for workaround:
+
+$ put [options] <key> -- <value>
+$ put [options] -- <key> <value>
+
+NOTES:
+
+This will error:
```

Comment:
@gyuho @heyitsanthony @jonboulle  Personally, I feel this example is too long. I feel the help message is good enough.